Sources/Logging/Locks.swift

```diff
@@ -125,7 +125,7 @@ extension Lock {
     /// - Parameter body: The block to execute while holding the lock.
     /// - Returns: The value returned by the block.
     @inlinable
-    internal func withLock<T>(_ body: () throws -> T) rethrows -> T {
+    func withLock<T>(_ body: () throws -> T) rethrows -> T {
```

Comment:
could we undo (or move to another PR) these formatting changes? makes the diff too large and harder to review

Reply:
I'll undo them; I have to stop using a pre-commit formatting hook to be honest; different versions of swift format, and projects not declaring all options which they rely on explicitly cause this :-(

Reply:
done.